notes/porting.creole

```diff
@@ -1,10 +1,17 @@
 ==Downloading Projects from Cloud
 The original source of these steps for downloading an existing project from the cloud compiler were posted to the mbed forum by Peter Drescher in [[http://developer.mbed.org/forum/mbed/topic/2701/?page=2#comment-14008|this post]].
 
-* Export the code on the cloud compiler and unpack into a new directory in the sample folder.
-* If you use an imported lib, open the .lib file. It will contain a link to get the library code.  Download the lib and unpack also.
-* Copy the makefile from the StdIO sample.
-* Change the project name inside the makefile.
-* Start BuildShell.
-* Go inside the project directory.
-* make clean all
\ No newline at end of file
+* Export the code on the cloud compiler
+ ** In the Program Workspace, [right-]click the project of interest (e.g. "mbed_blinky")
+ ** Choose "Export Program..." 
+ ** Select "GCC (ARM Embedded)", or perhaps "Zip Archive (with repositories)"
+ ** Unpack the project (e.g. "mbed_blinky") as a new directory in the "samples" folder.
+* If you use an imported lib,
+ ** Open the .lib file in an editor. It will contain a link to get the library code. 
```

Comment:
I have't had to do this step for a long time.  Usually I just delete the .lib files these days.  The source code for the libraries has typically already been pulled down (if I perform a "GCC (ARM Embedded)" export anyway).